src/pages/home/_trade-types.js

```diff
@@ -43,114 +20,61 @@ const TradingButton = styled(LinkButton)`
-                                'Get the best of both - the upside of margin trading with the simplicity of options.',
-                            )}
-                        </Text>
-                    </StyledCard>
-                </Container>
-            </Show.Desktop>
-            <Show.Mobile min_width="852">
-                <Container
-                    ai="center"
-                    direction="column"
-                    m="9rem auto 15rem !important"
-                    height="35rem"
-                    width="90% !important"
-                >
-                    <Flex direction="column">
-                        <Header align="center" size="3.2rem">
-                            {localize('Trade Types')}
-                        </Header>
-                        <Text align="center" size="2.4rem" mt="1.6rem" mb="5rem">
-                            {localize(
-                                'Trade what you like, the way you like it, and on your preferred market.',
-                            )}
-                        </Text>
-                    </Flex>
-                    <Flex>
-                        <StyledCard>
-                            <div>
-                                <MarginLogo />
-                            </div>
-                            <Header>{localize('Margin trading')}</Header>
-                            <Text>
-                                {localize(
-                                    'Trade with leverage and low spreads for better returns on successful trades.',
-                                )}
-                            </Text>
-                        </StyledCard>
-                        <StyledCard>
-                            <div>
-                                <OptionsLogo />
-                            </div>
-                            <Header>{localize('Options')}</Header>
-                            <Text>
-                                {localize(
-                                    'Earn fixed payouts by predicting an assets price movement within a fixed time.',
-                                )}
-                            </Text>
-                        </StyledCard>
-                        <StyledCard>
-                            <div>
-                                <MultipliersLogo />
-                            </div>
-                            <Header>{localize('Multipliers')}</Header>
-                            <Text>
-                                {localize(
-                                    'Get the best of both - the upside of margin trading with the simplicity of options.',
-                                )}
-                            </Text>
-                        </StyledCard>
-                    </Flex>
-                    <TradingButton mt="3rem" type="submit" secondary="true" to="/signup/">
+            <Container m="0 auto 8rem !important" height="35rem" width="90% !important">
```

Comment:
please dont' use important styling, you can extend container using styled component and pass in height and width to it. it will work without !important addition